docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -13,153 +13,235 @@ sidebar_position: 2
 
 ## Introduction
 
-Once deployed on a network, smart contracts cannot be changed. Their storage and logic **are set in stone**. If a smart contract is deployed with specific state variables and functions, these data and functionalities are there forever and cannot be altered.
+Externally Owned Accounts (EOAs) are the basic type of **account** in Ethereum controlled by a private key. If the private key is **compromised**, anyone can execute transactions from the account and access any assets it holds. As well, EOAs have no built-in mechanism for attaching any information or data, making it difficult to identify the person or entity using the account. They can only perform simple interactions using the **[CALL](https://www.evm.codes/#f1)** opcode and creating contracts using the **[CREATE](https://www.evm.codes/#f0)** opcode.
```

Comment:
Applying Grammarly + maybe linking to Ethereum.org for good external references and SEO.

```suggestion
[Externally Owned Accounts (EOAs)](https://ethereum.org/en/developers/docs/accounts/#externally-owned-accounts-and-key-pairs) are the primary type of **account** in Ethereum, controlled by a private key. If the private key is **compromised**, anyone can execute transactions from the account and access any assets it holds. EOAs have no built-in mechanism for attaching any information or data, making identifying the person or entity using the account challenging. They can only perform simple interactions using the **[CALL](https://www.evm.codes/#f1)** opcode and create contracts using the **[CREATE](https://www.evm.codes/#f0)** opcode.
```